File: app/src/main/assets/binary_to_dec.java
public class BinaryToDec {//-1
	public static void main(String[] args) {//2
      	Scanner scanner = new Scanner(System.in);//3,4
      	String bin ="";//3,4
      	System.out.print("Enter binary number: ");//-5
      	if (scanner.hasNextLine()) {//6
      	    bin = scanner.nextLine();//7
      	}//8,14,16,17
        int length = bin.length();//9
        int dec = 0;//-10
        for (int i = 0; i < length; i++) {//11
            int digit = bin.charAt (i) - '0';//12
            dec+=(digit*Math.pow(2,length-1-i));//13
        }//8,14,16,17
        System.out.printf("In decimal:%d\n",dec);//15
	}//8,14,16,17
}//8,14,16,17
//DESCRIPTION:
Create a Java method that takes a binary number as input from user and prints the decimal equivalent on the console.
Example 1:
Enter binary number: 101
In decimal: 5
Example 2:
Enter binary number: 11111
In decimal: 31

File: app/src/main/assets/binary_to_dec.cpp
int main (int argc, char *argv []) {//-1
    char binary [12];//2,3
    printf ("Enter binary number: ");//2,3
    scanf ("%s", binary);//4
    size_t length = strlen (binary);//5
    int decimal = 0;//-6
    for (int i = 0; i < length; i++) {//7
        int binaryDigit = binary[i] - '0';//8
        decimal += (binaryDigit * pow(2, length - 1 - i));//9
    }//10,13
    printf ("In decimal: %d\n", decimal);//11
    return 0;//12
}//10,13
//DESCRIPTION:
Create a C function that takes a binary number as input from user and prints the decimal equivalent on the console.
Example 1:
Enter binary number: 101
In decimal: 5
Example 2:
Enter binary number: 11111
In decimal: 31